components/autocomplete_selector.jsx

```diff
@@ -129,7 +129,7 @@ export default class AutocompleteSelector extends React.PureComponent {
                     <SuggestionBox
                         placeholder={placeholder}
                         ref={this.setSuggestionRef}
-                        listComponent={SuggestionList}
+                        listComponent={ModalSuggestionList}
```

Comment:
Is there anywhere else that we use the `AutocompleteSelector` that's outside of a modal?

Reply:
Yes, it is. It is used on ActionMenu component, which is used on the MessageAttachment component. I did not see that. I will solve this on the next commit.